app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+import shutil
+
+
+def copy_file(command: str) -> None:
+    path = command.split(" ")
+    if len(path) == 3:
+        with open(path[1], "r") as file_in, open(path[2], "w") as file_out:
+            if file_in == file_out:
+                pass
```

Comment:
why `pass`?
the user will not understand what went wrong, it is better to print something